### PR TITLE
[DOCS-352] Getting Started guide: users land on the Assets page

### PR DESCRIPTION
**Pull request by ana-dashuk-cobalt**
## Changelog

Once signed in, users land on the Assets page instead of the Pentests page.

### Added

| Page | Deploy Preview | Comment |
| ----- | ----- | ----- |
| Select the Pentest Type | https://deploy-preview-264--cobalt-docs.netlify.app/getting-started/select-pentest-type/ | Stage 2 |
| Review Your Asset | https://deploy-preview-264--cobalt-docs.netlify.app/getting-started/review-asset/ | Stage 3 |

### Updated

| Page | Deploy Preview | Comment |
| ----- | ----- | ----- |
| Define Your Assets | https://deploy-preview-264--cobalt-docs.netlify.app/getting-started/assets/ | Stage 1 |
| Set Pentest Requirements | https://deploy-preview-264--cobalt-docs.netlify.app/getting-started/pentest-objectives/ | Stage 4 |
| Specify Pentest Details | https://deploy-preview-264--cobalt-docs.netlify.app/getting-started/details/ | Stage 5 |
| Plan and Scope the Pentest | https://deploy-preview-264--cobalt-docs.netlify.app/getting-started/planning/ | Stage 6 |
| Review and Submit the Pentest | https://deploy-preview-264--cobalt-docs.netlify.app/getting-started/review-pentest/ | Stage 7 |

## Preview This Change

To see how this change looks in production, scroll down to **Deploy Preview**. Select the link that looks like `https://deploy-preview-<num>--cobalt-docs.netlify.app/`

## Variables

Help us support a “Write once, publish everywhere” single source of truth. If you see a line that looks like:

`{{% asset-categories %}}`

You’ve found a [shortcode](https://gohugo.io/content-management/shortcodes/) that we include in multiple documents.

You’ll find the content of the shortcode in the following directory:

https://github.com/cobalthq/cobalt-product-public-docs/tree/main/layouts/shortcodes

That shortcode has the same base name as what you see in the PR, such as `asset-categories.html`.

## Checklist for PR Author

[ ] Did you check for broken links and alt text?

Be sure to check for broken links and Alt Text issues. We have a partially automated process,
as described in this section of our repository README:
[Test Links and Alt Attributes](https://github.com/cobalthq/cobalt-product-public-docs/blob/main/README.md#test-links-and-alt-attributes).

### CODE REVIEW

File: content/en/Getting started/Assets/_index.md [hunk 1]
```diff
@@ -1,32 +1,24 @@
...
-Select how you want to proceed with your asset:
-
-- **Create a new asset**:
-  - On the **Asset** page, specify the [asset details](#asset-details). Once you select **Create Asset**, you land on the **Assets** page.
-  - To set up a pentest for this asset, select **Create Pentest** on the asset row.
-- **Use an existing asset**:
-  - Select an asset from the list. Once you select **Continue**, you can see asset details on the **Review Asset** screen. To update asset information, select **Edit Asset**.<br>
-  ![Review your asset in the pentest wizard](/gsg/ReviewAsset.png "Review your asset in the pentest wizard")
+- To add a single asset, select **New Asset**. Specify [asset details](#asset-details).
+- To upload assets in bulk in CSV or XLSX format, select **Bulk Assets**. Once uploaded, you can select an asset to add an image, [technology stack](/platform-deep-dive/assets/risk-advisories/#add-a-technology-stack-for-your-asset), and [attachments](/getting-started/assets/asset-description/#attachments).
```
Comment (mjang-cobalt) [inline content/en/Getting started/Assets/_index.md:19]: Do we specify how to format uploaded assets in these spreadsheet formats? Based on the description and the **Asset Details** section, I might conclude:

```csv
<asset title>, <asset type>, <asset description>
<asset title>, <asset type>, <asset description>
...
```

**Review by mjang-cobalt — CHANGES_REQUESTED** [review]
This is excellent work. Two more comments:

1. I see you've uploaded both sets of images to this PR. I prefer the latest, but if you still want to discuss with designers, I'm good with that too.
2. I think we'll need changes to this page https://deploy-preview-264--cobalt-docs.netlify.app/platform-deep-dive/pentests/

File: content/en/Getting started/Assets/_index.md (hunk 1 above)
Comment (ana-dashuk-cobalt) [reply content/en/Getting started/Assets/_index.md:19]: @mjang-cobalt In the overlay for uploading assets, users download a template where they can see the format.

Comment (ana-dashuk-cobalt) [reply content/en/Getting started/Assets/_index.md:19]: @mjang-cobalt With Danny's help, I updated the images displaying the pentest creation workflow.

Comment (mjang-cobalt) [reply content/en/Getting started/Assets/_index.md:19]: I'm going to "Resolve" this comment and suggest a new change, based on the "Assets_Upload_Template.xlsx" file from our app.

File: content/en/Getting started/Assets/_index.md [hunk 2]
```diff
@@ -1,32 +1,24 @@
...
+  - If the upload is successful, all your assets from the file are added. Otherwise, no assets are created. Our algorithm doesn't process the request partially.
+  - We don't prevent you from creating duplicate assets.
```
Comment (mjang-cobalt) [inline content/en/Getting started/Assets/_index.md:21]: - I'm suggesting that we include the Assets_Upload_Template.xlsx file to the `/static/gsg/` directory
- IMO, we don't need to specify what happens if the upload fails

```suggestion
- To upload assets in bulk in CSV or XLSX format, select **Bulk Assets**. To help, we have an [Assets Upload Template](/gsg/Assets_Upload_Template.xlsx).
   
   Once uploaded, you can select an asset to add an image, [technology stack](/platform-deep-dive/assets/risk-advisories/#add-a-technology-stack-for-your-asset), and [attachments](/getting-started/assets/asset-description/#attachments).
  - If the upload is successful, all your assets from the file are added.
  - We don't prevent you from creating duplicate assets.
```

Comment (ana-dashuk-cobalt) [reply content/en/Getting started/Assets/_index.md:21]: @mjang-cobalt The procedure describing how to upload assets in bulk was addressed in https://github.com/cobalthq/cobalt-product-public-docs/pull/203

The wording reflects exactly what Yogi requested back then. I don't think we should upload a template to Product Docs. The template may change. The SSoT is the template available in the UI.

Comment (mjang-cobalt) [reply content/en/Getting started/Assets/_index.md:21]: Thank you, I had forgotten that PR.

**Review by mjang-cobalt — APPROVED** [review]

**Review by yogipetkar — APPROVED** [review]
Changes look good.

**Pull request merged**